setup.cfg

```diff
@@ -37,6 +37,7 @@ install_requires =
   absl-py==1.4.0
   numpy>=1.23
   pandas>=2.0.1
+  tabulate==0.9.0
```

Comment:
Is it used anywhere else in the repo? Couldn't find anything with a quick search.